snapshotter/cmd/cc-snapshotter/main.go

```diff
@@ -0,0 +1,177 @@
+//go:build linux
+// +build linux
+
+/*
+   Copyright The containerd Authors.
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"net"
+
+	"google.golang.org/grpc"
+
+	snapshotsapi "github.com/containerd/containerd/api/services/snapshots/v1"
+	"github.com/containerd/containerd/contrib/snapshotservice"
+
+	"github.com/urfave/cli"
+
+	"cc-snapshotter/snapshotter"
+)
+
+const usage = "cc-snapshotter stores image layers as ext4 filesystems with dmverity information"
+
+func main() {
+	app := cli.NewApp()
+	app.Version = "v0.1"
+	app.Name = "cc-snapshotter"
+	app.Commands = []cli.Command {
+		cli.Command {
+			Name: "run",
```

Comment:
Is this the default action of a remote snapshotter? If so, what initiates the ``run`` command? Containerd?